sample-lbcd.conf

```diff
@@ -380,3 +380,5 @@
 ; be disabled if this option is not specified.  The profile information can be
 ; accessed at http://localhost:<profileport>/debug/pprof once running.
 ; profile=6061
+
+txindex=1
```

Comment:
We can revert this change now as line 300 already has a commented entry `txindex=1`, which should have been `txindex=0`.

The desired semantic in the sample-lbcd.conf is to have a commented entry showing default value for every setting. 


Since now we're enabling it, leaving that commented out `txindex=1`